components/CurrentOutput.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC } from 'react';
+import useSWR from 'swr';
+import NumberDisplay from './NumberDisplay';
+
+interface ActualOutputEntry {
+  datetime_utc: string;
+  actual_generation_kw: number;
+}
+
+function GetCurrentOutput(actual_outputs: ActualOutputEntry[]) {
+  var cur_time = Date.now();
+  var time = new Date(actual_outputs[0].datetime_utc).getTime();
+  var min = Math.abs(cur_time - time);
+  var index = 0;
+  for (var i = 1; i < actual_outputs.length; i++) {
+    time = new Date(actual_outputs[i].datetime_utc).getTime();
+    if (Math.abs(cur_time - time) < min) {
+      min = cur_time - time;
+      index = i;
+    }
+  }
+  return actual_outputs[index].actual_generation_kw;
+}
+
+const siteUUID = 'b97f68cd-50e0-49bb-a850-108d4a9f7b7e';
+
+const CurrentOutput = () => {
+  const fetcher = (url: string) => fetch(url).then((res) => res.json());
+  const { data: pv_actual } = useSWR(
+    `${process.env.NEXT_PUBLIC_API_BASE_URL}/api/sites/pv_actual/${siteUUID}`,
+    fetcher
+  );
+  console.log(pv_actual);
+
+  let cur_output = pv_actual
+    ? GetCurrentOutput(pv_actual?.pv_actual_values)
+    : null;
+
+  return (
+    <NumberDisplay
+      title="Current Output"
+      value={`${cur_output != null ? cur_output.toFixed(0) : 'Loading'} kW`}
```

Comment:
change loading here too for now

Reply:
actually let's make the value here say 'loading' if isLoading on the useSWR call is true, and then say 'N/A' if everything is loaded but the value is still falsy